src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -728,7 +728,9 @@ uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
     ret = mlx5dv_devx_general_cmd(md->super.dev.ibv_context, in, sizeof(in),
                                   out, sizeof(out));
     if (ret != 0) {
-        ucs_error("mlx5dv_devx_general_cmd(QUERY_HCA_CAP, ODP) failed: %m");
+        ucs_error("mlx5dv_devx_general_cmd(QUERY_HCA_CAP, ODP) failed,"
```

Comment:
can we add wrappers for devx calls like:
`ucs_status_t ucs_ib_mlx5_md_devx_general_cmd(ib_mlx5_md, const char *name, in, inlen, out,outlen)`
the wrapper will print the error

also - what about devx create/destroy obj?

Reply:
I've created wrappers for create/destroy/modify/general_cmd calls. These wrappers do not fit some cases (e.g. [here](https://github.com/openucx/ucx/blob/master/src/uct/ib/mlx5/dv/ib_mlx5dv_md.c#L202) and [here](https://github.com/openucx/ucx/blob/master/src/uct/ib/rdmacm/rdmacm_cm.c#L310))  but still reduces some code duplication.